linera-service/src/cli_wrappers/gcp_net.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{
+    util::ensure_connected_cluster_is_running, ClientWrapper, LineraNet, LineraNetConfig, Network,
+};
+use anyhow::Result;
+use async_trait::async_trait;
+use linera_base::data_types::Amount;
+use std::sync::Arc;
+use tempfile::{tempdir, TempDir};
+
+#[cfg(any(test, feature = "test"))]
+pub struct GCPNetTestingConfig;
```

Comment:
Can we replace `gcp` by `remote` everywhere? There is nothing specific about GCP.